contents/handbook/growth/sales/sales-operations.md

```diff
@@ -6,120 +6,66 @@ showTitle: true
+_Can I give a Scale customer a discount?_
 
-If an inbound email is about one of our Startup or Enterprise plans, you should create a Deal - more on this below. 
+Again, no need - we already have usage-based pricing which is _heavily_ discounted at higher volumes, and we only bill month-to-month, so customers don't need to feel locked in to a longer term contract. 
 
-In addition to hey@posthog.com and sales@posthog.com, we sometimes create special one-off email addresses to use for specific groups, such as for an event or promotion. If you create a Google group and you want messages to flow into HubSpot to be managed, make sure you add our [HubSpot inbox email address](hello-1@posthoginc.hs-inbox.com) to your group as a member.
+_How do I work with a customer who wants to sign an MSA?_
 
-### New PostHog signups
+This occasionally happens when we are dealing with very large companies, who may prefer to sign an MSA due to their internal procurement processes. We have a contract version of our standard terms and conditions that we can use for this - ask Charles. 
 
-All new users are automatically added via our Zapier app to the 'New PostHog User' stage of our Sales pipeline. Sorting these ensures that we can keep communication clear with a customer when they have multiple users on the account - it's really annoying for a customer if we are having parallel conversations with different people on their team!
+_How do I find out a customer's usage?_
 
-More on how we manage these users in the Deals section below. 
+[Go to this link](https://app.posthog.com/events?properties=%5B%7B%22key%22%3A%22users_who_logged_in__0__email%22%2C%22value%22%3A%22xyz%22%2C%22operator%22%3A%22icontains%22%2C%22type%22%3A%22event%22%7D%5D) and replace 'xyz' with the customer's company name. 
 
-### Manually adding new users
+_Can a customer transfer from self-hosted (e.g. Open Source) to Cloud?_
 
-You can also just manually add a user to HubSpot under 'Contacts'. When creating a new contact, try to add as much useful information as possible, especially about the type of company they work for and what their needs are. This enables us to provide them with the best possible experience. 
+Unfortunately we don't have a way to do this easily right now. If they have been on a Scale Paid plan, we can do this manually. If they are coming from the Open Source version, we suggest that they just restart on Cloud. 
 
-Once you have created a contact, you may want to add them to a Deal, depending on the context. 
+_Can a customer transfer from Cloud to Scale Paid?_
 
-Make sure you also assign someone as the Contact owner, so it's clear who is responsible for managing that relationship.
+Yes - we offer 2 months free of Scale Paid so we can run the migration in parallel. 
```

Comment:
3 months is fine with me / feels more reasonable. This isn't to run a migration, it's so they just run both systems in parallel - tracking events in _both_ places. That means when they switch off cloud they'll have 3 months of data to start with in the self hosted version.